blocks/header/header.css

```diff
@@ -42,10 +49,364 @@ header nav a:any-link {
+}
+
+header .nav-menu-overlay .nav-menu-column.left > ul {
+  padding: 0;
+}
+
+header .nav-menu-overlay .nav-menu-column.left ul {
+  font-family: var(--body-font-family);
+  font-size: 24px;
+  font-weight: 700;
+  line-height: 38.13px;
+  margin: 0;
+  text-align: left;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .nav-menu-overlay-back {
+  position: absolute;
+  left: 30px;
+  padding: 0;
+  margin: 0;
+  z-index: 2;
+  top: 28px;
+  background-color: white;
+  height: 44px;
+  width: 44px;
+  border-radius: 50%;
+  
+}
+
+header .nav-menu-overlay .nav-menu-column.right .highlight-link a{
+color: white;
+font-weight: 800;
+
+
+}
+
+header .nav-menu-overlay .nav-menu-column.left ul li span::before {
+  content: "";
+  width: 7px;
+  height: 2px;
+  top: 15px;
+  border-radius: 20px;
+  position: absolute;
+  left: 15px;
+  background-color: #169af3;
+  transform: rotate(45deg);
+  border: 2px;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .highlight-link a span::before{
+  content: "";
+  width: 12px;
+  display: inline-block;
+  height: 3px;
+  margin-bottom: 3px;
+  margin-left: 8px;
+  border-radius: 10px;
+  background-color: white;
+  border: 1px;
+}
+
+header .nav-menu-overlay .nav-menu-column.left ul li span::after {
+  content: "";
+  width: 7px;
+  height: 2px;
+  top: 20px;
+  left: 15px;
+  position: absolute;
+  background-color: #169af3;
+  border-radius: 20px;
+  transform: rotate(-45deg);
+  border: 2px;
+}
+  
+header .nav-menu-overlay .nav-menu-column.right .highlight-link span::after
+{
+  content: "";
+  width: 1px;
+  margin-left: -6px;
+  padding: 2px;
+  height: 1px;
+  display: inline-block;
+  transform: rotate(225deg);
+  border-bottom: 3px solid white;
+  border-left: 3px solid white;
+  border-radius: 1px;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .nav-menu-overlay-back::before {
+  content: "";
+  width: 3px;
+  position: absolute;
+  left: 13px;
+  padding: 3px;
+  height: 3px;
+  top: 14px;
+  transform: rotate(45deg);
+  border-bottom: 3px solid black;
+  border-left: 3px solid black;
+  border-radius: 1px;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .browse-country{
+  width: 90%;
+  display: flex;
+  flex-direction: row;
+position: relative;
+  justify-content: start;
+  align-items: center;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .browse-country .country-list{
+  width: 280px;
+  top: 40px;
+  left: -160px;
+  z-index: 2;
+  background-color:  #001C2D;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .browse-country .country-list:has(a:hover){
+display: block !important;
+}
+
+header .nav-menu-overlay .nav-menu-column.right .browse-country div{
+ position: relative;
+ 
+}
+
+header .nav-menu-overlay .nav-menu-column.left ul li span {
+  display: inline-block;
+  min-width: 36px;
+  min-height: 36px;
+  border-radius: 50%;
+  position: relative;
+  margin-right: 15px;
+  margin-bottom: -8px;
```

Comment:
try to avoid negative margins as much as possible. Only use if there's no other way.